docs/SRS-Volere/SRS.tex

```diff
@@ -252,22 +252,52 @@ \subsection{Products That Can Be Copied}
 
 \section{New Problems}
 \subsection{Effects on the Current Environment}
-\lips
+\begin{enumerate} [{}1.]
```

Comment:
why is [{}1.] needed here

Reply:
Removes the indent, aligns it with the other lists. I removed them.